dash-renderer/src/persistence.js

```diff
@@ -265,10 +265,22 @@ const noopTransform = {
     apply: (storedValue, _propValue) => storedValue,
 };
 
-const getTransform = (element, propName, propPart) =>
-    propPart
-        ? element.persistenceTransforms[propName][propPart]
-        : noopTransform;
+const getTransform = (element, propName, propPart) => {
+    if (typeof element.persistenceTransforms !== 'undefined') {
+        if (
+            Object.getOwnPropertyNames(element.persistenceTransforms).includes(
```

Comment:
Can be simplified to `if(element.persistenceTransforms[propName])` in this case

Reply:
91c5324